cmd/nitro/nitro.go

```diff
@@ -312,34 +312,34 @@ func mainImpl() int {
 	var l1TransactionOptsValidator *bind.TransactOpts
 	var l1TransactionOptsBatchPoster *bind.TransactOpts
 	sequencerNeedsKey := nodeConfig.Node.Sequencer.Enable && !nodeConfig.Node.Feed.Output.DisableSigning
```

Comment:
```suggestion
	sequencerNeedsKey := (nodeConfig.Node.Sequencer.Enable && !nodeConfig.Node.Feed.Output.DisableSigning) || nodeConfig.Node.BatchPoster.Enable
```
Super minor, but I think we can add the batch poster here and then we can remove the duplication from the two checks below. Then we could also define a `validatorNeedsKey = nodeConfig.Node.Staker.OnlyCreateWalletContract || validatorCanAct` or something like that as well to dedup that. Other than that looks good to me

Reply:
nice suggestion, done.